scripts/releasePlugin.sc

```diff
@@ -0,0 +1,63 @@
+import $ivy.`com.lihaoyi::requests:0.6.5`
+import $ivy.`com.lihaoyi::upickle:0.9.5`
+
+@main
+def release(
```

Comment:
This script contains logic to publish the zipped plugin to github under the release.